src/styles/login.scss

```diff
@@ -0,0 +1,6 @@
+$color: rgb(255,0,0);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=6016116)